office365_connector.py

```diff
@@ -2394,115 +3276,128 @@ def handle_action(self, param):
 
         self.debug_print("action_id", self.get_action_identifier())
 
-        if action_id == 'test_connectivity':
+        if action_id == "test_connectivity":
```

Comment:
I'm not sure I understand the point of changing all of the formatting from single to double quotes here: Python standards say to use single quotes unless there is a single quote character in the string itself, which it won't ever be.

Reply:
Hey @tonyc-phantom 
The changes were made by the external contributor, therefore haven't changed them.